src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { TodoType } from '../../types/TodoType';
+
+export enum SortType {
+  All = 'All',
+  Active = 'Active',
+  Completed = 'Completed',
+}
+
+type Props = {
+  onSortType: (newSortType: SortType) => void;
+  selectedSortType: SortType;
+  itemsLeft: TodoType[];
+};
+
+const sortTypes = [SortType.All, SortType.Active, SortType.Completed];
+
+export const Footer: React.FC<Props> = ({
+  onSortType,
+  selectedSortType,
+  itemsLeft,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${itemsLeft.length} items left`}
+      </span>
+      <nav className="filter">
+        {sortTypes.map(sortType => (
+          <li key={Math.random()} style={{ listStyle: 'none' }}>
```

Comment:
Don't use math random for the key. You can create an array of objects instead
Smth like this 
```
const sortTypes = [
  { id: 1, type: SortType.All },
  ...
]